README.md

```diff
@@ -136,7 +148,13 @@ The following represent a few of the applications that teams developed over the
 
 - [DFINITY Explorer](https://github.com/dfinityexplorer/dfinityexplorer-dashboard) - DFINITY Explorer is an open-source network dashboard built by the DFINITY community.
 
-- [Babmeo](https://github.com/BerkeleyBlockchain/dfinity-research) - Babmeo is an image & video sharing site with transparent recommendations that integrates with DFINITY’s LinkedUp, BigMap, and BigSearch projects to recommend, store, and search videos.
+- [Babmeo](https://github.com/BerkeleyBlockchain/dfinity-research) - Babmeo is an image & video sharing site with transparent recommendations that integrates with DFINITY's LinkedUp, BigMap, and BigSearch projects to recommend, store, and search videos.
+
+- [Bookworm](https://github.com/rstout/bookworm) - An open publishing application inspired by China Literature.
+
+- [Shield](https://github.com/crusso/shield) - An app connecting at-risk people with nearby volunteers.
+
+- [Hero](https://github.com/stanleygjones/hackathon) - A random "killer app" generator inspired by the Wu-Tang Name Generator.
```

Comment:
Duplicate links are not allowed and were found by the linting tool. Since this section was duplicated with some duplicate links and also some different links, I merged the two sections together.